go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/jarcoal/httpmock v1.3.1
 	github.com/joho/godotenv v1.5.1
 	github.com/jpillora/longestcommon v0.0.0-20161227235612-adb9d91ee629
+	github.com/json-iterator/go v1.1.12
 	github.com/keboola/go-client v1.26.5
```

Comment:
HTTP source is using faster JSON implementation,
same as in the  [go-client](https://github.com/keboola/go-client/blob/main/pkg/client/json.go).